tests/integration/components/sl-chart-test.js

```diff
@@ -0,0 +1,117 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent( 'sl-chart', 'Integration | Component | sl chart', {
+    integration: true
+});
+
+test( 'Defaults applied correctly', function( assert ) {
+    this.set( 'testseries', []);
+    this.set( 'testoptions', {});
+
+    this.render( hbs`
+      {{#sl-chart series=testseries options=testoptions}}
+      {{/sl-chart}}
+    ` );
```

Comment:
https://github.com/softlayer/sl-ember-components/pull/669/files So I went off how @juwara0 made his spacing in his render statements in that PR. Not exactly sure how this is wrong.